src/components/AddItem/AddItem.jsx

```diff
@@ -1,3 +1,168 @@
+import styles from "./AddItem.module.scss";
+import { useState, useRef } from "react";
+
 export default function AddItem() {
-  return <div>addItem</div>;
+  const [imagePreview, setimagePreview] = useState(null);
+  const [formData, setFormData] = useState({
+    image: "",
+    name: "",
+    about: "",
+    price: "",
+    tag: [],
+  });
+  const inputRef = useRef();
+  const active = Object.values(formData).every((value) =>
+    Array.isArray(value) ? value.length > 0 : value
+  );
+
+  function handleChange(e) {
+    const { name, value } = e.target;
+    if (name === "image") {
+      const newImage = e.target.files[0];
+      setimagePreview(URL.createObjectURL(newImage));
+      setFormData((prev) => ({ ...prev, image: newImage }));
+    } else {
+      setFormData((prev) => ({ ...prev, [name]: value }));
+    }
+  }
+  function handleTag(e) {
+    if (e.key === "Enter") {
+      e.preventDefault(); //폼 내부에서 엔터 -> 자동 제출되는거 막음
+      setTimeout(() => {
+        const newValue = inputRef.current.value.trim();
+        if (newValue) {
+          setFormData((prev) => ({
+            ...prev,
+            tag: [...prev.tag, newValue],
+          }));
+        }
+        inputRef.current.value = "";
+      }, 0);
+    }
+  }
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(formData); // 서버 전송 등 처리
+  };
+
+  console.log(formData);
+  function handleRemoveTag(tagToRemove) {
+    setFormData((prev) => ({
+      ...prev,
+      tag: prev.tag.filter((tag) => tag !== tagToRemove),
+    }));
+  }
+
+  return (
+    <>
+      <form className={styles.form} onSubmit={handleSubmit}>
```

Comment:
❗️ 수정요청
form 하나만 반환하고 있어서 Fragment로 감쌀 필요가 없을 것 같아요.

```suggestion
    <form className={styles.form} onSubmit={handleSubmit}>
```